packages/trace-viewer/src/sw/snapshotRenderer.ts

```diff
@@ -618,6 +618,21 @@ function rewriteURLsInStyleSheetForCustomProtocol(text: string): string {
   });
 }
 
+// url() inside a <style> tag can mess up with html parsing, so we encode some of them.
+// As an example, the following url will close the </style> tag:
+// url('data:image/svg+xml,<svg><defs><style>.a{fill:none}</style></defs><g class="a"></g></svg>')
+const urlToEscapeRegex1 = /url\(\s*'([^']*)'\s*\)/ig;
+const urlToEscapeRegex2 = /url\(\s*"([^"]*)"\s*\)/ig;
+function escapeURLsInStyleSheet(text: string): string {
+  const replacer = (match: string, url: string) => {
+    // Conservatively encode only urls with a closing tag.
```

Comment:
Why conservatively? Either the encoding works inside of `<style>` or it doesn't.

Reply:
I'd like to be on the safe side and alter the actual content as less a possible. We know that `</` urls are broken anyway, so fixing them is definitely a win.